### PR TITLE
updated folder Structure

### PR DESCRIPTION
**Pull request by vrindavnair**

### CODE REVIEW

File: src/pages/adminPages/FranchiseDetailsPages/FranchiseProfileDetails.jsx
Comment (JaseelaAlimahamed) [inline src/pages/adminPages/FranchiseDetailsPages/FranchiseProfileDetails.jsx:1]: make folder name first letter small

File: src/pages/adminPages/CategorySubCategoryPages/AddCategoryPage.jsx
Comment (JaseelaAlimahamed) [inline src/pages/adminPages/CategorySubCategoryPages/AddCategoryPage.jsx:1]: make folder name first letter small

**Review by JaseelaAlimahamed — CHANGES_REQUESTED** [review]

File: src/pages/adminPages/FranchiseDetailsPages/FranchiseProfileDetails.jsx
Comment (vrindavnair) [reply src/pages/adminPages/FranchiseDetailsPages/FranchiseProfileDetails.jsx:1]: implemented as requested

File: src/pages/adminPages/CategorySubCategoryPages/AddCategoryPage.jsx
Comment (vrindavnair) [reply src/pages/adminPages/CategorySubCategoryPages/AddCategoryPage.jsx:1]: implemented as requested

**Pull request closed without merging**